templates/index.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<div class="jumbotron">
+    <h1>Project 451</h1>
+
+    <p class="lead"></p>
+
+    {% if not user %}
```

Comment:
I think it can be more pretty if we format this if conditions such as:
```
    {% if not user %}
        <p><a class="btn btn-lg btn-success" href="/join" role="button">Sign up for free today</a></p>
    {% else %}
        <p><a class="btn btn-lg btn-success" href="/mailbox" role="button">Go to system!</a></p>
    {% endif %}
```